### PR TITLE
libupm: update to 2.0.0

### PR DESCRIPTION
**Pull request by nxhack**
Maintainer: @blogic, me
 Compile tested: ar71xx, LEDE trunk r10251-bec8fb1
 Run tested: NONE

Description:
 update to 2.0.0
add PKG_BUILD_PARALLEL

https://github.com/openwrt/packages/issues/9254

Signed-off-by: Hirokazu MORIKAWA <morikw2@gmail.com>

### CODE REVIEW

File: libs/libupm/Makefile [hunk 1]
```diff
@@ -51,13 +51,14 @@ UPM_MODULES:= \
...
 # (require libbacnet) tb7300 t8100 e50hx bacnetmstp
+# (require libtinyb) 2jciebu01_ble 2jciebu01_usb
 
 define Package/libupm/Default
   SECTION:=libs
   CATEGORY:=Libraries
   SUBMENU:=IoT
   URL:=https://github.com/intel-iot-devkit/upm
-  DEPENDS:=+libmraa +librt
+  DEPENDS:=@!arc @!armeb @!powerpc +libmraa +librt
```
Comment (neheb) [inline libs/libupm/Makefile]: Those dependencies are implicit because of libmraa.

File: libs/libupm/Makefile [hunk 2]
```diff
@@ -8,21 +8,23 @@
...
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/intel-iot-devkit/upm/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=b253742b01146f4b86e20714aa24e75ca1e9d46629eab9aa8db070ce94cf3619
+PKG_HASH:=7dd2f4165b71e071d100b58d6a392f3cf57b0f257c82ffabf49e931b5ed6bc23
 PKG_BUILD_DIR:=$(BUILD_DIR)/upm-$(PKG_VERSION)
 
+CMAKE_INSTALL:=1
+CMAKE_BINARY_SUBDIR:=build
+PKG_USE_MIPS16:=0
+PKG_BUILD_PARALLEL:=1
```
Comment (neheb) [inline libs/libupm/Makefile]: I'd like all of these below PKG_LICENSE_FILES to be consistent with other packages.

File: libs/libupm/Makefile (hunk 1 above)
Comment (nxhack) [reply libs/libupm/Makefile]: fix

File: libs/libupm/Makefile (hunk 2 above)
Comment (nxhack) [reply libs/libupm/Makefile]: fix

**Pull request merged**